Makefile

```diff
@@ -48,27 +56,25 @@ clean-dockerdemo:
 ssh:
 	@vagrant ssh netplugin-node1 || echo 'Please run "make demo"'
 
-unit-test: build
-	CONTIV_HOST_GOPATH=$(GOPATH) CONTIV_HOST_GOBIN=$(HOST_GOBIN) \
```

Comment:
I may be I am missing something. I just see two mounts there, one for the `./` directory (i.e. the dir where Vagrantfile resides)  and other for `./bin` directory.

However, `CONTIV_HOST_GOPATH` used to mount the $GOPATH of the host itself which will automatically mount the `$GOPATH/src` and `$GOPATH/bin` dirs and hence make godep and golint available in the vm as well.